app/assets/main/react_components/routes/lifestyle_footprints/footprint_form_container/components/footprint_form/components/result_page/components/category_section/CategoryPage.jsx

```diff
@@ -14,12 +14,14 @@ const CategoryPage = ({ footprint }) => {
 
   return (
     <>
-      <Title customStyle="text-lgr" text={title} />
+      <Title custom_style="text-lgr" text={title} />
```

Comment:
Detta blev fel vid pull project-2, det är customStyle som används. Så den ändringen skedde i min förra styling-PR :) 
```suggestion
      <Title customStyle="text-lgr" text={title} />
```

Reply:
Sjukt namngett av mig!